third_party/proton/test/test_cmd.py

```diff
@@ -49,12 +49,14 @@ def test_instrument_exec():
             result.append(line.split())
 
     if is_hip():
-        assert [row[0] for row in result] == ['0', '1', '2', '3']
-        assert [row[1] for row in result] == ['matmul_kernel', 'matmul_kernel', 'matmul_kernel', 'matmul_kernel']
-        assert [row[2] for row in result
-                ] == ['instrument.py:32:20', 'instrument.py:33:20', 'instrument.py:32:20', 'instrument.py:33:20']
-        assert [row[3] for row in result] == ['SHARED', 'SHARED', 'SHARED', 'SHARED']
-        assert [row[4] for row in result] == ['STORE', 'STORE', 'LOAD', 'LOAD']
+        assert len(result) == 7
```

Comment:
@CRobeck: FYI about this change--it looks we were missing support for `llvm.intr.masked.{load|store}` in `--instrument=print-mem-spaces` before?

Reply:
I don't think we were missing support exactly they were just rolled into a single load/store so the analysis here didn't capture as many. Now it looks like some of what were masked ops are stand alone.

Reply:
Okay, I'm not familiar with internals about `--instrument=print-mem-spaces`. Though previously we don't print any `GLOBAL` memory ops at all?

Reply:
Oh, I see. Yes, you're right. That pass just walks the instructions and looks for LLVM IR Load/Stores. So we would have had to explicitly look for:
```
Intrinsic::masked_load
Intrinsic::masked_store
```
to support those. OK, no harm removing them then.